json-schema/json-schema/src/test/scala/endpoints4s/TuplerTests.scala

```diff
@@ -5,18 +5,45 @@ class TuplerTests {
   def tupling[A, B](a: A, b: B)(implicit tupler: Tupler[A, B]): tupler.Out =
     tupler(a, b)
 
-  def forall[A, B, C, D, E, F](a: A, b: B, c: C, d: D, e: E, f: F): Unit = {
+  def forall[A, B, C, D, E, F, G, H, I](a: A, b: B, c: C, d: D, e: E, f: F, g: G, h: H, i: I): Unit = {
+    tupling((), a): A
+    tupling(a, ()): A
     tupling(a, b): (A, B)
-    tupling(a, (b, c)): (A, B, C)
     tupling((a, b), c): (A, B, C)
+    tupling(a, (b, c)): (A, B, C)
+    tupling((a, b, c), d)  : (A, B, C, D)
     tupling((a, b), (c, d)): (A, B, C, D)
-    tupling(a, (b, c, d, e)): (A, B, C, D, E)
+    tupling(a, (b, c, d))  : (A, (B, C, D))
+    tupling((a, b, c, d), e)  : (A, B, C, D, E)
+    tupling((a, b, c), (d, e)): (A, B, C, (D, E))
     tupling((a, b), (c, d, e)): (A, B, C, D, E)
-    tupling((a, b, c), d): (A, B, C, D)
-    tupling((a, b, c, d), e): (A, B, C, D, E)
-    tupling((a, b, c, d, e), f): (A, B, C, D, E, F)
-    tupling(a, ()): A
-    tupling((), a): A
+    tupling(a, (b, c, d, e))  : (A, (B, C, D, E))
```

Comment:
We should get a flat tuple of type `(A, B, C, D, E)` here. The line you deleted from `Tupler.scala` should produce such a result.

Reply:
Yep, confirmed. I'll not break this with further work.